src/components/main-site/HeaderPhoto.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import "bootstrap/dist/css/bootstrap.min.css";
+import { withStyles } from "@material-ui/styles";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faUserCircle } from "@fortawesome/free-solid-svg-icons";
+
+import UPEclass from "../../assets/img/UPEclass.jpg"
+
+const styles = {
+    masthead: {
+        marginBottom: "50px",
+        background: "no-repeat center center",
+        backgroundColor: "#868e96",
+        backgroundAttachment: "scroll",
+        position: "relative",
+        backgroundSize: "cover",
+        backgroundImage: `url(${UPEclass})`,
+        
+      },
+      mastheadSiteHeading: {
+        padding: "350px 0 150px",
+        color: "#fff",
+        "@media only screen and (min-width:768px)": {
+          padding: "325px 0!important",
+        },
+      },
+}
+
+
```

Comment:
Can we tighten the whitespace in this file? Rule of thumb is no more than 1 empty line of whitespace between code blocks